libs/utils/decorate.js

```diff
@@ -17,12 +17,12 @@ export function decorateButtons(el, size) {
       parent.insertAdjacentElement('afterend', button);
       parent.remove();
     }
+    const actionArea = button.closest('p, div');
```

Comment:
This seems like it could assign `actionArea` to the wrong el since closest will keep going up the tree till it finds one of those.  Though I see it's existing code so I guess it's guaranteed that `p, div` will be within the aside?

Reply:
@chrischrischris The snippet to add the actionArea class to the parent of a action button was already working in decorate.js for all blocks. 
Previously the actionArea class was added only to the parent of the first button that was encountered in the block. The snippet was moved as in case there is a block with multiple action areas, then their parent elements will get 'action-area' class added to them.